lte/gateway/fabfile.py

```diff
@@ -532,33 +532,29 @@ def run_with_retry(c_gw, command, retries=10):
 
 
 @task
-def get_test_summaries(
-        c,
-        dst_path="/tmp",
-        integration_tests=False,
-        sudo_tests=False,
-        dev_vm_name="magma",
-):
-    c.run('mkdir -p ' + dst_path)
+def get_test_summaries(c, sudo_tests=False, integration_tests=False):
```

Comment:
There are several issues with this function at the moment:

1. In line 541 you misspelled `sudo_tests` as `sudo_test`. 
2. Previously, if we specified `integration_tests=True` and `sudo_tests=True` we would call `_get_test_summaries_from_vm` twice and retrieve both sets of test results. In the new implementation we would enter both `if` statements in 547-548 and then only retrieve the integration test results.
3. I believe the `if` statements in 547-548 should both have a newline for after the colon.
4. See @LKreutzer 's comment
5. The condition in 541 could be somewhat simplified by using `if not(sudo_tests or integration_tests)`, but that might be a matter of preference.
6. I don't quite understand why we want to remove the `dst_path` argument.

Reply:
1. Changed 
2. We separated sudo tests and integration tests some time ago. A workflow now only executes one of them.
3. Changed
4. Changed
5. Changed
6. The `dest_path` was always the same in CI, which makes it irrelevant to be parsed as an argument. Now the results are just collected in the `test_results` folder.